roles/postgresql/README.md

```diff
@@ -13,19 +13,33 @@ Role Variables
 
 ### Connection information to the database server: 
 ```ini
-  postgres_host - postgres server
-  postgres_port - defaults to postgres default port
-  postgres_admin_user - defaults to none
-  postgres_admin_password - defaults to none
-  postgresql_is_local - should always be false unless testing
+  postgresadmin: "postgres"
+  db_clusteradmin_user: "postgres"
+  db_clusteradmin_password: "{{ vault_postgres_admin_password }}"
+  postgres_admin_password: '{{ vault_postgres_admin_password }}'
+  postgres_port: 5432
+  postgres_admin_user: "{{ postgresadmin }}"
+  pg_hba_contype: "host"
+  pg_hba_method: "md5"
+  pg_hba_postgresql_database: "all"
+  pg_hba_postgresql_user: "all"
+  pg_hba_source: "{{ ansible_host }}/32"
+  postgresql_is_local: true  # rarely true unless one is testing creating a new db server
+  postgres_version: 15  # on server side this will install the postgresql server version
```

Comment:
Could we use separate variables for server-side and client-side? That would make the variables and the docs more clear.